uber/templates/groups/form.html

```diff
@@ -25,147 +25,194 @@
 
     $(function() {
         {% if group.is_new and not new_dealer %}
-            $.field("status").find("[value={{ UNAPPROVED }}]").text("n/a (not a paid Dealer)");
+            $.field("status").find("[value={{ c.UNAPPROVED }}]").text("n/a (not a paid Dealer)");
         {% endif %}
     });
 </script>
 
+<div class="panel panel-default">
+
 <h2> Group Info </h2>
 
 <div style="text-align:center">
     <a href="index">Return to group list</a>
 </div>
 
-<form method="post" action="form">
+<form method="post" action="form" class="form-horizontal">
 {% csrf_token %}
 <input type="hidden" name="id" value="{{ group.db_id }}" />
 <input type="hidden" name="new_dealer" value="{{ new_dealer }}" />
 
-<table>
-<tr>
-    <td width="25%"> <b><nobr>Name of Group:</nobr></b> </td>
-    <td width="75%"> <input class="focus" type="text" name="name" value="{{ group.name }}" /> </td>
-</tr>
-<!-- we need to register a placeholder group leader with this new group -->
+<div class="form-group">
+    <label class="col-sm-2 control-label">Name of Group</label>
+    <div class="col-sm-6">
+        <input class="focus" type="text" name="name" value="{{ group.name }}" />
+    </div>
+</div>
+
+<!-- we need to register a placeholder group leader for new dealers -->
 {% if new_dealer %}
-    <tr>
-        <td> <nobr><b>Owner's First and Last Name:</b></nobr> </td>
-        <td>
+    <div class="form-group">
+        <label class="col-sm-2 control-label">Owner's First and Last Name</label>
+        <div class="col-sm-6">
             <input type="text" name="first_name" value="{{ first_name }}" style="width:10em" />
             <input type="text" name="last_name" value="{{ last_name }}" style="width:15em" />
-        </td>
-    </tr>
-    <tr>
-        <td> <nobr><b>Owner's Email Address:</b></nobr> </td>
-        <td> <input type="text" name="email" value="{{ email }}" /> </td>
-    </tr>
+        </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-2 control-label">Owner's Email Address</label>
+        <div class="col-sm-6">
+            <input type="text" name="email" value="{{ email }}" />
+        </div>
+    </div>
 {% endif %}
-<tr>
-    <td> <b>Tables:</b> </td>
-    <td>
+
+<div class="form-group">
+    <label class="col-sm-2 control-label">Tables</label>
+    <div class="col-sm-6">
         <select name="tables">
-            {% options ADMIN_TABLE_OPTS group.tables %}
+            {% int_options 0 9 group.tables %}
```

Comment:
Why did you put in hardcoded integer values here? It's not uncommon to sell half-tables to Dealers - I think even MAGFest may do it.